File: design-patterns/java/DesignPatterns/src/ar/edu/unlam/cdp/patterns/command/Button.java
package ar.edu.unlam.cdp.patterns.command;

public class Button {

	private Command command;
	
	public Button(Command command) {
		this.command = command;
	}
	
	// Other button stuff :)
	
	public void click(){
		this.command.execute();
	}
	
}

File: design-patterns/cs/Patrones/Patrones/Command/Button.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Patrones.Command
{
    public class Button
    {
        private Command command;

        public Button(Command command)
        {
            this.command = command;
        }

        public void Click()
        {
            this.command.Execute();
        }
    }
}
